ExampleCodes/FFTW/Basic/main.cpp

```diff
@@ -1,5 +1,7 @@
 #include <AMReX.H>
 
+#include <cmath>
+
```

Comment:
I don't think this is needed (compiles for me fine without it) - cmath probably is included via something else included.